cmd/siac/walletcmd.go

```diff
@@ -209,9 +210,11 @@ use it instead of displaying the typical interactive prompt.`,
 
 const askPasswordText = "We need to encrypt the new data using the current wallet password, please provide: "
 
-const currentPasswordText = "Current Password: "
-const newPasswordText = "New Password: "
-const confirmPasswordText = "Confirm: "
+const (
+	currentPasswordText = "Current Password: "
+	newPasswordText     = "New Password: "
+	confirmPasswordText = "Confirm: "
+)
```

Comment:
In general PRs (and especially commits) shouldn't make unnecessary stylistic changes. If we want to tweak the style, we can apply it project-wide in a separate PR.

Reply:
My lint-on-save picked this one up. I guess I can edit the file in `nano` and revert it without the auto-linter overriding me.

Reply:
might as well leave it in this case, but yeah going forward I'd rather not have random changes added by your linter

Reply:
no promises that I won't miss some, but I'll check in the future.